softdevice_controller/README.rst

```diff
@@ -43,14 +43,19 @@ Variants for the Arm Cortex-M33 processor are available as soft-float only.
 | Periodic Advertising Sync      |                 |              | X         |
 | Transfer - Sender              |                 |              |           |
 +--------------------------------+-----------------+--------------+-----------+
+| Periodic Advertising Sync      |                 |              | X         |
+| Transfer - Receiver            |                 |              |           |
++--------------------------------+-----------------+--------------+-----------+
 
 .. note::
    The following limitations apply to the listed features:
 
    * For Connectionless CTE Advertising, angle of arrival (AoA) is supported, but angle of departure (AoD) is not.
    * For Connection CTE Response, angle of arrival (AoA) is supported, but angle of departure (AoD) is not.
-   * For the LE Power Control Request feature, the support is experimental, and acceptable power reduction (APR), controller-initiated power level management are not supported.
+   * For the LE Power Control Request feature, the support is experimental.
    * For Periodic Advertising Sync Transfer - Sender, the support is experimental.
+   * For Periodic Advertising Sync Transfer - Receiver, the support is experimental, and only one sync transfer reception may be in progress at any one time per link.
+     After using LE HCI Set Default Periodic Advertising Sync Transfer Parameters, there must be a periodic sync available for every new connection established.
```

Comment:
Does that mean a periodic sync context needs to be available for every connection?

Reply:
@alexstanoev-nordic , maybe you can add some insight to this?

Reply:
Yes - but only when using LE HCI Set Default Periodic Advertising Sync Transfer Parameters to set the default state for all new connections to accept PAST.